### ISSUE TITLE
Ability to ignore certain drift reports

### ISSUE THREAD

**Issue by moltar** [cdklabs/cdk-drift-monitor#673]
There are many resources that produce false positive drift results. Even RDS often does this with parameter groups. Would be great to be able to ignore-list some of the known drifts.

**Comment by rgoltz** [comment]
I agree with this valid request - I've added a request for CloudFormation Team here: https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1533 (to get an option to define & exclude dedicated resources/properties from CFN Drift Detection)
